kasa/iot/iotdevice.py

```diff
@@ -278,7 +278,9 @@ def supported_modules(self) -> list[str | ModuleName[Module]]:
         """Return a set of modules supported by the device."""
         # TODO: this should rather be called `features`, but we don't want to break
         #       the API now. Maybe just deprecate it and point the users to use this?
```

Comment:
```suggestion
```
Probably not needed anymore :-)

Reply:
Deprecated the whole property in favour of `modules` now that that property returns only the supported ones.